src/__internal__/label/label.style.js

```diff
@@ -60,23 +56,21 @@ export const StyledLabelContainer = styled.div`
     css`
       ::after {
         content: "(optional)";
-        font-weight: 350;
+        font-weight: 350; //TODO: (tokens) use token var(--fontWeights400)
```

Comment:
Why can't we do this now?

Reply:
In DS we use 3 font-weight variants: (400, 500, 700). In this step we replace the theme with tokens only. I've given a TODO here to introduce font-weight tokens in the next iteration, when we adjust the components for the new DS.